ports/esp32s2/boards/microdev_micro_s2/mpconfigboard.h

```diff
@@ -28,7 +28,7 @@
 #define MICROPY_HW_BOARD_NAME       "microS2"
 #define MICROPY_HW_MCU_NAME         "ESP32S2"
 
-#define MICROPY_HW_LED (&pin_GPIO21)
+#define MICROPY_HW_LED_STATUS (&pin_GPIO21)
 #define MICROPY_HW_BUTTON (&pin_GPIO0)
```

Comment:
```suggestion
```
Reason for the removal:
- `NeoPixel` is used for status so no need for led.
- `MICROPY_HW_BUTTON` definition is not used anywhere.

Reply:
Oh, I just forgot that, thanks!